src/ttkbootstrap/widgets.py

```diff
@@ -96,7 +95,7 @@ class DateEntry(ttk.Frame):
     def __init__(
         self,
         master=None,
-        dateformat=locale.nl_langinfo(locale.D_FMT),
+        dateformat=r"%Y-%m-%d",
```

Comment:
What happens if you use `%x`? Does it show the correct date format for your locality?
https://docs.python.org/3.8/library/datetime.html#strftime-and-strptime-format-codes

Reply:
`dateformat=r"%x"`

Yes indeed, it worked. The default pt_BR format:
03/02/2022 (day/month/year)

I think you better make this change, then also modify the documentation with the necessary explanations.